internal/app/dogma/cli_cmds.go

```diff
@@ -228,8 +228,21 @@ func CLICommands() []cli.Command {
 			},
 		},
 		{
-			Name:      "watch",
-			Usage:     "Watches a file in the path",
+			Name:  "watch",
+			Usage: "Watches a file in the path",
+			Description: `Watch events are printed to stdout by default.
+   You can customize this behavior by using --listener <executable> option.
+   The command you specified as <executable> can read updated content body through its STDIN.
+   Other meta data about the watch event are available via environment variables below.
+
+     DOGMA_WATCH_EVENT_PATH - The path to the file you're watching
+     DOGMA_WATCH_EVENT_CONTENT_TYPE - The content type of the file, JSON or TEXT
+     DOGMA_WATCH_EVENT_REV - The revision number of the watch event
+     DOGMA_WATCH_EVENT_URL - The URL of the target file
+
+   e.g.
+     # Print foo.json content when it gets updated
+     dogma watch --listener cat /pj/repo/foo.json`,
 			ArgsUsage: "<project_name>/<repository_name>/<path>",
 			Flags:     []cli.Flag{revisionFlag, jsonPathFlag, streamingFlag, listenerFlag},
 			Action: func(c *cli.Context) error {
```

Comment:
nit: By the way, this file is mixed with tab and space though it is not related to this PR. Could you change the tabs to spaces, please?😉

Reply:
Actually I tried to use tabs first but it broke help text indentation because `commandHelpTemplate` (https://github.com/line/centraldogma-go/pull/42/commits/c9b577e794349fa14c5acee334f67df96001b027#diff-33d65ee15203de0d14a825149497009c0697ad35c123552cd1a7f573c9958257L71) is indented using spaces.

like this
<img width="705" alt="screen shot 2021-07-15 16 05 33" src="https://user-images.githubusercontent.com/21692/125744325-4c108592-0f46-46b0-a5c1-30cc6bc8364c.png">

Reply:
I didn't know the details. Let's keep it as it is.